packages/oscal-react-library/src/components/OSCALCatalogGroup.js

```diff
@@ -42,16 +42,38 @@ const StyledControlDescriptionWrapper = styled("div")`
   padding: 1em;
 `;
 
+/**
+ * Validates fragment and determines if top fragment item matches a list ID.
+ *
+ * @param {*} urlFragment The fragment currently being handled
+ * @param {*} previousHandledFragment The previous handled fragment
+ * @param {*} fragmentSuffix String with current item and sub items
+ * @param {*} listId The group/control ID
+ * @returns
+ */
```

Comment:
I don't want to hold this issue more than necessary, so I won't ask for TypeScript, but I also really hope that some of these parameters can't just be anything. Can we be more specific?

Reply:
Good point, I rushed this JSDoc and missed some details.